### DISCUSSION TITLE
Where all starts - Soroban submisson

### DISCUSSION THREAD

**Discussion by Xicaaa** [stellar/sorobanathon#129] [General]
QUEST 0 - WHERE ALL STARTS
 https://litemint.com/items/GCP3C4FRYOT7XMXMOFZNAILQSM4MBJF3RMG5KP3VKVCKRQ2QJDWWPGIZ/CHAPTER0 

QUEST 1 - THE TEMPLE
 https://litemint.com/items/GAWGLFYA3L3DZV3G3TJFBSZ63CIQS2ZHPW5KJSIHKQ3Y4T3YQ5CSW5GL/CHAPTER1

QUEST 2 - LAVA SNAKES
https://litemint.com/items/GADEG4KYV3S3M3JWYWHRVMEWZWVWGYUVMCCORNXGG2ARJBRDFFPNBR4L/CHAPTER2

QUEST 3 - WOODEN DOOR
 https://litemint.com/items/GDT4JXQ3ELBHTE4C7YSQ5Z2PPGRU37OIZK3W6LP6CZQGF7YBY5RJVM4R/CHAPTER3

QUEST 4 - THE CIRCULAR ROOM
 https://litemint.com/items/GABXRZHS3EWN75GQQU3ELNAITPWZSUAQLLTKTAJBYSPWG7P4AEQMDEQT/CHAPTER4

QUEST 5 - THE DARK CAVERN
https://litemint.com/items/GB3BKSH7QKZGJCMGSR7IWDUO6XS7ZGBBNDIYCSK7OAU6FW2YJSYIPVYD/CHAPTER5

QUEST 6 - THE GOLDEN SOROBAN
 https://litemint.com/items/GARX5JJ6XOKIHRFMZLCGNBHDQ3NZ743OK5OHXP6J3U4YICEHRQLMOU34/CHAPTER6

Created by https://twitter.com/xicaaa__

**Comment by Xicaaa** [comment]
#NFT

**Comment by stellarjohn** [comment]
Hi [Xicaaa](https://github.com/Xicaaa)!

Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
